fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -556,6 +556,10 @@ private void addColumnInternal(OlapTable olapTable, Column newColumn, ColumnPosi
                 throw new DdlException("Can not assign aggregation method on column in Duplicate data model table: " + newColName);
             }
             if (!newColumn.isKey()) {
```

Comment:
why not put "from lint 559 to line 562" outside of if stmt?

Reply:
I didn't understand what you mean...